Lib/idlelib/filelist.py

```diff
@@ -66,7 +69,20 @@ def unregister_maybe_terminate(self, edit):
             del self.dict[key]
         del self.inversedict[edit]
         if not self.inversedict:
-            self.root.quit()
+            try:
+                clip = self.root.clipboard_get().encode('utf-16')
+                if clip and sys.platform[:3] == 'win' and \
+                   len(clip) <= io.DEFAULT_BUFFER_SIZE:
+                    # Avoid exceeding the pipe bufsize.
+                    with subprocess.Popen('clip', shell=True,
```

Comment:
I added it because, without it I found that a console window flashes temporarily (this may just be because I am on a slower computer and there also may be a better way of resolving it).

Reply:
I saw flashes with one of my other experiments, but never with this.  A mute point for the moment.  Anything we do should be tried on multiple machines.